Objects/Converters/ConverterRevit/ConverterRevitShared/ConversionUtils.cs

```diff
@@ -1299,5 +1299,50 @@ private string RemoveProhibitedCharacters(string s)
         return s;
       return Regex.Replace(s, "[\\[\\]{}|;<>?`~]", "");
     }
+
+    // MEGA HACK to get the slope arrow of a roof which is technically not accessable by the api
+    // https://forums.autodesk.com/t5/revit-api-forum/access-parameters-of-slope-arrow/td-p/8134470
+    private void GetSlopeArrowHack(ElementId elementId, out Point tail, out Point head, out double tailOffset, out double headOffset, out double slope)
+    {
+      List<ElementId> deleted = null;
+      tail = null;
+      head = null;
+      tailOffset = 0;
+      headOffset = 0;
+      slope = 0;
+      using (var t = new Transaction(Doc, "TTT"))
+      {
+        t.Start();
+        deleted = Doc.Delete(elementId).ToList();
+        t.RollBack();
+      }
+      foreach (ElementId id in deleted)
+      {
+        ModelLine l = Doc.GetElement(id) as ModelLine;
+        if (l == null) continue;
+        if (!l.Name.Equals("Slope Arrow")) continue; // TODO: does this work with other languages of Revit?
```

Comment:
You can change language with some command line args like this: https://knowledge.autodesk.com/support/revit/troubleshooting/caas/CloudHelp/cloudhelp/2021/ENU/Revit-Installation/files/GUID-BD09C1B4-5520-475D-BE7E-773642EEBD6C-htm.html